combat-trainer.lic

```diff
@@ -213,6 +213,9 @@ class LootProcess
     @ritual_type = thanatology['ritual_type'].downcase
     echo("  @ritual_type: #{@ritual_type}") if $debug_mode_ct
 
+    @cycle_rituals = false
+    @cycle_rituals = @ritual_type == "cycle"
```

Comment:
Well, because it's not initiated before. If it's not cycle it's not initiated or nil? I'm not sure the interactions with nil so i figured false would be better.